tests/e2e/specs/dashboard/surveys.test.js

```diff
@@ -63,6 +63,10 @@ describe( 'dashboard surveys', () => {
 	it( 'shows a survey', async () => {
 		await visitAdminPage( 'admin.php', 'page=googlesitekit-dashboard' );
 
+		// Wait for 6 seconds before checking whether we see the survey or not
+		// because surveys are shown only in 5 seconds after rendering the page.
+		await new Promise( ( resolve ) => setTimeout( resolve, 6000 ) );
```

Comment:
Let's add an assertion here that we _don't_ see a survey in the first 5 seconds (maybe 3 or 4 to be safe)? Then assertion below could get an added [`timeout`](https://github.com/argos-ci/jest-puppeteer/blob/main/packages/expect-puppeteer/README.md#expectinstancetomatchelementselector-options) of a few seconds as a grace period for it to appear within. Otherwise this test would pass as-is even if the survey wasn't delayed from showing.

Reply:
Yep, good point. Updated.